test/nn/test_multihead_attention.py

```diff
@@ -0,0 +1,690 @@
+# Owner(s): ["module: nn"]
+import contextlib
+import random
+import unittest
+import unittest.mock as mock
+import collections
```

Comment:
Lintrunner does correctly fail for unused import: https://github.com/pytorch/pytorch/actions/runs/3562653891/jobs/5984583096#step:7:29